src/Domains.js

```diff
@@ -263,6 +263,7 @@ export const genericTlds = [
     "coupon", // coupon Amazon Registry Services, Inc.
     "coupons", // coupons Black Island, LLC
     "courses", // courses OPEN UNIVERSITIES AUSTRALIA PTY LTD
+    "cpa", // cpa American Institute of Certified Public Accountants

```

Comment:
better to copy the whole array, with the comments that show the last update date.